pkg/storage/system_intake.go

```diff
@@ -634,3 +634,26 @@ func (s *Store) UpdateSystemIntakeLinkedCedarSystem(ctx context.Context, id uuid
 
 	return s.FetchSystemIntakeByID(ctx, id)
 }
+
+// FetchRelatedSystemIntakes retrieves System Intakes that share a CEDAR system ID and/or a contract
+// number with a given System Intake
+func (s *Store) FetchRelatedSystemIntakes(ctx context.Context, id uuid.UUID) ([]*models.SystemIntake, error) {
+	intakes := []*models.SystemIntake{}
+
+	query := `
+		SELECT intakes_a.*
```

Comment:
Should this be selecting `intakes_b.*`? I ask because when I make a query locally, it seems to be returning the ID I asked for, not the ID that's related to it 🤔

Reply:
![image](https://user-images.githubusercontent.com/15203744/187904612-c5eee4a3-e174-4a10-bd92-346db71672d9.png)

Reply:
Similarly, if there's multiple related systems, I get multiple copies of the intake referenced in the ID

![image](https://user-images.githubusercontent.com/15203744/187904759-dcddfcc3-8755-4adf-bc97-14c6b4014326.png)

Reply:
Ah yes, the intakes_a.* was a last minute typo I made when renaming the tables in the query, whoops